src/mmw/js/src/modeling/templates/controls/utils.html

```diff
@@ -1,7 +1,7 @@
 {% macro thumb(label, value, size=56) %}
     <li>
         <div class="thumb" data-value="{{ value }}">
-            <svg>
+            <svg height="{{ size }}" width="{{ size }}">
```

Comment:
Without the height/width attributes I added, the svg was much wider than the rect and div (.thumb) and when the mouse was over the svg, it would propagate mouseenter events to the div, which is not what we want.